components/export/api/src/main/java/org/phenotips/export/internal/DataToCellConverter.java

```diff
@@ -69,13 +69,13 @@ public void phenotypeSetup(Set<String> enabledFields) throws Exception
         String sectionName = "phenotype";
         String[] fieldIds = { "phenotype", "phenotype_code", "phenotype_combined", "phenotype_code_meta",
             "phenotype_meta", "negative_phenotype", "negative_phenotype_code", "negative_phenotype_combined",
-            "phenotype_by_section" };
+        "phenotype_by_section" };
         // FIXME These will not work properly in different configurations
         String[][] headerIds =
-            { { "phenotype", "positive" }, { "code", "positive" }, { "phenotype", "code", "positive" },
-                { "meta_code", "phenotype", "positive" }, { "meta", "phenotype", "positive" },
-                { "negative", "phenotype" }, { "negative", "code" }, { "negative", "code", "phenotype" },
-                { "category" } };
+        { { "phenotype", "positive" }, { "code", "positive" }, { "phenotype", "code", "positive" },
+            { "meta_code", "phenotype", "positive" }, { "meta", "phenotype", "positive" },
+            { "negative", "phenotype" }, { "negative", "code" }, { "negative", "code", "phenotype" },
+            { "category" } };
```

Comment:
It looks like the only change here is whitespace -- if that's the case, I'd say the change should be discarded.